src/app/features/courses/components/courses-search/courses-search.component.ts

```diff
@@ -9,11 +9,17 @@ export class CoursesSearchComponent implements OnInit {
   @Input()
   searchValue: string = '';
 
+  @Output()
+  searchValueChange = new EventEmitter<string>();
+
+  @Output()
+  searchClickEvent = new EventEmitter();
+
   constructor() {}
 
   ngOnInit(): void {}
 
   searchClick() {
```

Comment:
Rename method to `onSearch`. SearchClick sounds a little bit strange and not functional.